src/main/java/uk/gov/justice/hmpps/prison/service/digitalwarrant/DigitalWarrantService.kt

```diff
@@ -221,4 +222,30 @@ class DigitalWarrantService(
       ).id
     }
   }
+  @Transactional
```

Comment:
The convention is to mark the service class as @Transactional(readonly=true) and override update/create functions with @Transactional